src/builder/builder_test.go

```diff
@@ -11,18 +11,19 @@ import (
 func TestStatefulsetBuilder(t *testing.T) {
 	t.Run("Simple, no modifiers", func(t *testing.T) {
 		b := GenericBuilder[mocks.DataMock]{}
-		actual := b.Build()
+		actual, err := b.Build()
+		assert.NoError(t, err)
 		expected := mocks.DataMock{}
 		assert.Equal(t, expected, actual)
 	})
 	t.Run("One modifier", func(t *testing.T) {
 		b := GenericBuilder[mocks.DataMock]{}
 
 		modifierMock := mocks.NewModifierMock[mocks.DataMock]()
-		modifierMock.On("Modify", mock.Anything).Return()
+		modifierMock.On("Modify", mock.Anything).Return(nil)
 		modifierMock.On("Enabled").Return(true)
 
-		actual := b.AddModifier(modifierMock).Build()
+		actual, _ := b.AddModifier(modifierMock).Build()
```

Comment:
Why not add
```suggestion
		actual, err := b.AddModifier(modifierMock).Build()
		require.NoError(t, err)
```
for this occasions.